File: 0075-sort-colors/0075-sort-colors.java
class Solution {
    public void sortColors(int[] nums) {
        int N = nums.length;
        int tmp;
        boolean flag;
        for(int re = 0; re < N; re++) {
            flag = false;
            for(int i = 0; i < N - 1; i++) {
                if(nums[i] > nums[i + 1]) {
                    flag = true;
                    
                    tmp = nums[i];
                    nums[i] = nums[i + 1];
                    nums[i + 1] = tmp;
                }
            }
            
            if(!flag) break;
        }
    }
}

File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        flag = False
        for re in range(len(nums)):
            flag = False
            for i in range(len(nums) - 1):
                if nums[i] > nums[i + 1]:
                    nums[i], nums[i + 1] = nums[i + 1], nums[i]
                    flag = True
                
            if flag == False:
                break
        